app/src/main/java/ar/com/wolox/android/training/network/IRest.java

```diff
@@ -0,0 +1,25 @@
+package ar.com.wolox.android.training.network;
+
+import com.google.gson.JsonElement;
+
+import java.util.Map;
+
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.HeaderMap;
+import retrofit2.http.Query;
+
+/**
+ * IRest services
+ */
+public interface IRest {
+
+    @GET("/users")
+    Call<JsonElement> getUsersListRequest(@HeaderMap Map<String, String> headers);
```

Comment:
I missed this. Here you should return a list of users. I'll explain it now in person.